src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PAYMENT = 40;
+  const BIGGER_DISCONT = 7;
```

Comment:
The constant name 'BIGGER_DISCONT' is not descriptive enough to explain what it represents. Consider renaming it to 'LONG_TERM_RENTAL_DAYS' to better reflect its purpose.